config/testconfig.yaml

```diff
@@ -8,10 +8,10 @@ general:
   tnf_entry_point_script: "run-tnf-container.sh"
   # Example
   # tnf_config_dir: "/Users/bmandal/rhdev/github.com/cnfcert-tests-verification/tnf_config"
-  tnf_config_dir: "/path/to/cnfcert-tests-verification/tnf-config"
```

Comment:
personally-specific testconfig changes again. Based on some quick Google-ing this could be a use case for git's assume-unchanged flag: https://practicalgit.com/blog/make-git-ignore-local-changes-to-tracked-files.html